### PR TITLE
Dsa hashAlgs -> hashAlg, and pqg

### PR DESCRIPTION
**Pull request by Kritner**
Spec updates related to #334, #335, #337, #338

### CODE REVIEW

**Comment by celic** [comment]
Thanks for handling this @Kritner, this looks ready to merge @atvassilev

File: artifacts/acvp_sub_dsa.html [hunk 1]
```diff
@@ -1673,7 +1709,7 @@ <h1 id="rfc.appendix.A.1">
       ],
       "L": 2048,
       "N": 256,
-      "HashAlgs": [
+      "HashAlg": [
```
Comment (bfussell) [inline artifacts/acvp_sub_dsa.html]: HashAlg or hashAlg  ? I think we use the latter in most algorithms

File: artifacts/acvp_sub_dsa.html [hunk 2]
```diff
@@ -1691,7 +1727,7 @@ <h1 id="rfc.appendix.A.1">
       ],
       "L": 3072,
```
Comment (bfussell) [inline artifacts/acvp_sub_dsa.html]: L , N or l, n ? Seems like the implementation is using lower case for both.

**Comment by celic** [comment]
All the examples should be the agreed camelCase.

**Comment by bfussell** [comment]
I was looking at an outdated PR. I deleted my comments, everything looks good.

**Pull request merged**